tests/test_routes.py

```diff
@@ -27,4 +29,56 @@ def test_add_like_missing_card(client):
     # raise Exception("Complete test with assertion about response body")
     assert response_body == {
         "message": "Card with id 1 was not found."
-    }
\ No newline at end of file
+    }
+
+
+def test_add_like_invalid_card(client):
+    # Act
+    response = client.patch("/cards/something/add_like")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 400
+    assert response_body == {"message": f"something is invalid"}
+
+
+def test_create_one_card(client, one_board):
+
+    response = client.post("/boards/1/cards", json={
+        "likes_count": 0,
+        "message": "Test card",
+    })
+    response_body = response.get_json()
+
+    assert response.status_code == 201
+    assert response_body == {
+        "board_id": 1,
+        "card_id": 1,
+        "likes_count": 0,
+        "message": "Test card"
+    }
+
+    new_card = Card.query.get(1)
+    assert new_card
+    assert new_card.message == "Test card"
+    assert new_card.likes_count == 0
+
+
+# def test_post_task_ids_to_goal(client, one_goal, three_tasks):
```

Comment:
Did you mean to leave the commented part below in?

Reply:
looks like this is an outdated version 
![image](https://github.com/dorl9039/back-end-inspiration-board/assets/43712089/8329dc4e-5a7c-40dc-b6a5-d83e7254f855)
 this comment doesn't appear in the latest changes:

Reply:
Sorry about that!